inc/switchtec/switchtec.h

```diff
@@ -53,6 +53,11 @@ struct switchtec_dev;
 #define SWITCHTEC_UNBOUND_PORT 255
 #define SWITCHTEC_PFF_PORT_VEP 100
 
+#define SWITCHTEC_BOOT_PART_START (0xa8000000)
+#define SWITCHTEC_MAP0_PART_START (0xa8020000)
+#define SWITCHTEC_MAP1_PART_START (0xa8040000)
+#define SWITCHTEC_PART_LEN (0x10000)
```

Comment:
Nit: you don't need parentheses around numbers in defines (only around expressions).

Reply:
Thanks.